site/content/getting-started/_index.md

```diff
@@ -1,237 +1,188 @@
+Recreate your kind cluster, following the guide but using the following Operator-compatible config file:
 ```yaml
 kind: Cluster
 apiVersion: kind.x-k8s.io/v1alpha4
 nodes:
 - role: control-plane
 - role: worker
   extraPortMappings:
-  - containerPort: 80
+  - containerPort: 30080
     hostPort: 80
-    listenAddress: "0.0.0.0"  
-  - containerPort: 443
+    listenAddress: "0.0.0.0"
+  - containerPort: 30443
     hostPort: 443
     listenAddress: "0.0.0.0"
-``` 
-	    
-### Create a Kubernetes cluster using kind:  
+```
 
-Create a cluster using kind.config.yaml file  
+Install the Contour Operator & Contour CRDs:
 
-```yaml
-$ kind create cluster --config=kind.config.yaml
+```bash
+$ kubectl apply -f https://projectcontour.io/quickstart/operator.yaml
 ```
 
-Verify the nodes are ready by running:  
+Verify the Operator deployment is available:
 
-```yaml
-$ kubectl get no
-```  
+```bash
+$ kubectl get deploy -n contour-operator
+NAME               READY   UP-TO-DATE   AVAILABLE   AGE
+contour-operator   1/1     1            1           1m
+```
 
-You should see a 2 nodes listed with status **Ready**:  
-- kind-control-plane
-- kind-worker
+Install an instance of the `Contour` custom resource:
 
-Congratulations, you have created your cluster environment. You're ready to install Contour.  
- 
-## 2. Install Contour and Envoy
-Run the following to install Contour:
+```bash
+# If using a kind cluster
+kubectl apply -f https://raw.githubusercontent.com/projectcontour/contour-operator/{{< param latest_version >}}/examples/contour/contour-nodeport.yaml
 
-```yaml
-$ kubectl apply -f https://projectcontour.io/quickstart/contour.yaml
+# If using a cluster with support for services of type LoadBalancer
+kubectl apply -f https://raw.githubusercontent.com/projectcontour/contour-operator/{{< param latest_version >}}/examples/contour/contour.yaml
+```
+
+Verify the `Contour` custom resource is available (it may take up to several minutes to become available):
+
+```bash
+$ kubectl get contour/contour-sample
+NAME             READY   REASON
+contour-sample   True    ContourAvailable
 ```
 
 Verify the Contour pods are ready by running the following: 
 
-```yaml
+```bash
 $ kubectl get pods -n projectcontour -o wide
-```  
+```
+
 You should see the following:
 - 2 Contour pods with each status **Running** and 1/1 **Ready**  
-- 1 Envoy pod with the status **Running** and 1/1 **Ready**  
+- 1+ Envoy pod(s), each with the status **Running** and 1/1 **Ready**  
+ 
+## Test it out!
 
 Congratulations, you have installed Contour and Envoy! Let's install a web application workload and get some traffic flowing to the backend.
- 
-## 3. Install a kuard workload (simple application)  
-This section installs [kuard][9] to verify web traffic is flowing to the pods through envoy.
 
+TODO is there a different sample workload then?
```

Comment:
Need to resolve this; given the next line says not to expose this publicly we probably want some other workload.

Reply:
Maybe one of the echoservers or `httpbin`? It just needs to show enough to validate the thing works, not be fancy.

Reply:
Easy enough, switched it over.